domains/client/block-preprocessor/src/lib.rs

```diff
@@ -451,7 +453,7 @@ where
     ) -> Vec<Block::Extrinsic> {
         exts.into_iter()
             .filter(|ext| {
-                match verify_xdm::<CClient, CBlock, Block, _>(
+                match is_valid_xdm::<CClient, CBlock, Block, _>(
```

Comment:
`verify_xdm` makes more sense to me since `is_valid_xdm` imply it should return `bool` instead of `Result`

Reply:
It is returning bool. The result due to the runtime sp_api::Error which we cannot avoid.